plugins/exec/store.ml

```diff
@@ -0,0 +1,21 @@
+type 'a t = { next_key : Uint63.t ref; table : (Uint63.t, 'a) Hashtbl.t }
+
+let unwrap : Constr.t -> Uint63.t = fun x ->
+  match Constr.kind x with
+  | Constr.Int x -> x
+  | _ -> assert false
+
+let create _ =
+  { next_key = ref Uint63.zero; table = Hashtbl.create ~random:false 100 }
```

Comment:
This would be far more efficient to use an extensible array here!

Reply:
I will create an issue, and let you do your magic (: